docs/commerce/commerce.md

```diff
@@ -6,28 +6,43 @@ page_type: landing_page
 
 # Commerce
 
-The commerce component of [[= product_name =]] covers various steps of making a transaction,
-from listing available products, through adding products to a cart, to checkout and confirmation.
+The commerce component of [[= product_name =]] covers various areas of managing an e-commerce presence: from configuring payment and shipping methods, through processing a transaction (listing available products, adding products to a cart, processing checkout and sending confirmation), all the way to order management.
 
 [[= cards([
 "commerce/cart/cart",
-"commerce/cart/cart_api",
 "commerce/checkout/checkout",
-"commerce/checkout/configure_checkout",
-"commerce/checkout/checkout_api",
-"commerce/checkout/customize_checkout",
 "commerce/order_management/order_management",
-"commerce/order_management/configure_order_management",
-"commerce/order_management/order_management_api",
 "commerce/payment/payment",
+"commerce/shipping_management/shipping_management",
+"commerce/storefront/storefront"
+], columns=4) =]]
+
+## Configure
+
+[[= cards([
+"commerce/checkout/configure_checkout",
+"commerce/order_management/configure_order_management",
 "commerce/payment/configure_payment",
+"commerce/shipping_management/configure_shipment",
+"commerce/storefront/configure_storefront"
+], columns=4) =]]
+
+## Extend
+
+[[= cards([
+"commerce/checkout/customize_checkout",
+"commerce/payment/extend_payment",
+"commerce/storefront/extend_storefront"
+], columns=4) =]]
+
+## Explore API reference
```

Comment:
It is not an API reference, I would suggest Explore Commerce API. This title as well as [Personalization API reference](https://doc.ibexa.co/en/latest/personalization/api_reference/api_reference/) are shown very high in search results and may confuse readers. Let's be specific.